backend/app/Console/Kernel.php

```diff
@@ -10,7 +10,6 @@ class Kernel extends ConsoleKernel
     /**
      * Define the application's command schedule.
      *
-     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
```

Comment:
You might want to undo the changes to these kinds of files since the files changed is increasing more than necessary.